app/models/concerns/task_tree_render_module.rb

```diff
@@ -29,9 +36,10 @@ def self.new_renderer # rubocop:disable all
       )
       ttr.config.default_atts = [:id, :status, :ASGN_BY, :ASGN_TO, :updated_at]
       ttr.config.heading_label_template = lambda { |appeal|
-        docket = (defined?(appeal.docket_type) && appeal.docket_type) ||
-                 (defined?(appeal.docket_name) && appeal.docket_name)
-        "#{appeal.class.name} #{appeal.id} (#{docket}) "
+        docket = appeal.docket_name.first.titleize
+        appeal_type = appeal.is_a?(Appeal) ? appeal.stream_type : appeal.type
```

Comment:
`Appeal#type` works here as well! 

https://github.com/department-of-veterans-affairs/caseflow/blob/15de1d3e26cff73a716e308d0e10d6df09d88e9e/app%2Fmodels%2Fappeal.rb#L94-L96
```suggestion
        appeal_type = appeal.type
```